Makefile

```diff
@@ -24,3 +24,28 @@ start_all:
 
 start_all_secure:
 	cd tools && ./start_all_tmux.sh secure
+
+start_test_all:
+	cd tools && chmod +x start_test.sh && ./start_test.sh
+
+clean_all_service:
+	cd lucida \
+	&& cd calendar && make clean \
+	&& cd ../commandcenter && make clean \
+	&& cd ../djinntonic && make clean \
+	&& cd ../imagematching/opencv_imm && make clean \
+	&& cd ../../musicservice && make clean \
+	&& cd ../questionanswering/OpenEphyra && make clean \
+	&& cd ../../weather && make clean \
+	&& cd ../..
```

Comment:
I think you should be able to `cd lucida && make clean` instead of having to go into each directory right?